tools/script_template.py

```diff
@@ -1,5 +1,29 @@
 import argparse
 import yaml
+import openmc
+import numpy as np
+
+def open_flux_file(flux_file):
+    with open(flux_file, 'r') as flux_data:
+        flux_lines = flux_data.readlines()
+    return flux_lines
+
+def parse_flux_lines(flux_lines):
+    '''
+    Uses provided list of flux lines and group structure applied to the run to create an array of flux entries, with:
+    # rows = # of intervals = total # flux entries / # group structure bins
+    # columns = # group structure bins
+    input : flux_lines (list of lines from ALARA flux file)
+    output : flux_array (numpy array of shape # intervals x number of energy groups)
+    '''
+    energy_bins = openmc.mgxs.GROUP_STRUCTURES['VITAMIN-J-175']           
+    all_entries = np.array(' '.join(flux_lines).split(), dtype=float)
```

Comment:
There should probably be some way to make sure that the flux file is not empty:

```suggestion
    all_entries = np.array(' '.join(flux_lines).split(), dtype=float)
    if len(all_entries) == 0:
        raise Exception("The chosen flux file is empty.")
```